tests/apps_test.go

```diff
@@ -108,6 +108,7 @@ func appsLogsTest(t *testing.T, params *utils.DeisTestConfig) {
 
 func appsOpenTest(t *testing.T, params *utils.DeisTestConfig) {
 	utils.CurlApp(t, *params)
+	CurlWithFail(t, fmt.Sprintf("http://%s.%s", "this-app-does-not-exist", cfg.Domain), true, "404 Not Found")
```

Comment:
The function is located in the `utils` package, so you need to call `utils.CurlWithFail`. Also, you need to add the `fmt` package to the top of this file and change `cfg` to `params`. Thanks!